crates/cast/tests/e2e/declare.rs

```diff
@@ -118,10 +118,13 @@ fn scarb_build_fails(contract_path: &str, accounts_file_path: &str) {
     let snapbox = Command::new(cargo_bin!("sncast"))
         .current_dir(CONTRACTS_DIR.to_string() + contract_path)
         .args(args);
+    let assert = snapbox.assert().success();
+    let stderr_output = String::from_utf8(assert.get_output().stderr.clone()).unwrap();
 
-    snapbox.assert().success().stderr_matches(indoc! {r#"
-        error: scarb build returned non-zero exit code: 1
-    "#});
+    assert!(
+        stderr_output.contains("error: Scarb build returned non-zero exit code: 1"),
+        "Expected error message not found in stderr: {stderr_output}",
```

Comment:
Should we start adding messages to all the `assert` statements?

Reply:
I found it very handy for this test, as by default failed test result did not provide much information:

```
---- e2e::declare::scarb_build_fails::when_cairo_2_and_scarb_toml_does_not_exist stdout ----
thread 'e2e::declare::scarb_build_fails::when_cairo_2_and_scarb_toml_does_not_exist' panicked at 'assertion failed: stderr_output.contains(\"error: Scarb build returned non-zero exit codee: 1\")', crates/cast/tests/e2e/declare.rs:124:5
```

Sometimes for asserts there is expected vs returned comparison, so in such cases it is not necessary. But, in scenarios like this one I'd say yes - we should add more context to errors.

Reply:
Could you please create an issue then? 🙏 

Comparison between expected and actual output is provided if we use `snapbox.assert().stderr_matches`

Reply:
Sure, https://github.com/foundry-rs/starknet-foundry/issues/353 :+1: